### PR TITLE
Add helper text to manage roles in event overview

### LINKED ISSUE #893: Add helper text to manage roles in event overview
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
-->

**Expected behavior:**
image

### PR DESCRIPTION
**Pull request by ritikamotwani**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
Helper text appears in the manage roles. 
image

Fixes #893

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-894-open-event-frontend-fossasia.surge.sh

File: app/templates/components/events/view/overview/manage-roles.hbs [hunk 1]
```diff
@@ -33,9 +33,9 @@
             </div>
           </td>
         </tr>
+      {{else}}<div class="ui basic segment">{{t 'No records to show'}}</div>
```
Comment (CosmicCoder96) [inline app/templates/components/events/view/overview/manage-roles.hbs]: please put this in a separate line like 
```hbs
{{else}}
  <div class="ui basic segment">
    {{t 'No records to show'}}
  </div>
{{/each}}
```

**Comment by niranjan94** [comment]
Deployment link: https://pr-894-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-894-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-894-open-event-frontend-fossasia.surge.sh

**Pull request merged**